### PR TITLE
{data,lib}[foss/2016b,intel/2016b] psycopg2 v2.7, PostgreSQL v9.6.2 w/ Python 2.7.12

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2569.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b994bb0699d0d3334c4a9ab4176e2828 for a full test report.

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.6.2-foss-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+homepage = 'https://www.postgresql.org/'
+description = """PostgreSQL is a powerful, open source object-relational database system.
+ It is fully ACID compliant, has full support for foreign keys,
+ joins, views, triggers, and stored procedures (in multiple languages).
+ It includes most SQL:2008 data types, including INTEGER,
+ NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.
+ It also supports storage of binary large objects, including pictures,
+ sounds, or video. It has native programming interfaces for C/C++, Java,
+ .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation."""
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.6.2-foss-2016b-Python-2.7.12.eb]: a bit long?

Comment (boegel) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.6.2-foss-2016b-Python-2.7.12.eb]: maybe, but I'm just using the same description as in the other `PostgreSQL` easyconfigs

If this is too long, we should change it consistently.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2054.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b638ceb7dde0d6d333c65da1afc846b2 for a full test report.

**Pull request merged**